src/pages/Vehicle/Power/Power.tsx

```diff
@@ -0,0 +1,18 @@
+import {DataSection} from "./components/DataSection.tsx";
+import {PowerState} from "./models/PowerState.ts";
+import './Power.scss'
+
+type Props = {state: PowerState}
+
+export const Power = (props: Props) => {
+
+    const state = props.state
+
+    const renderSections = () => state.getSections().map((it, index) => <DataSection key={index} section={it} />)
+
+    return (
+        <div className={'power-section'}>
+            {renderSections()}
```

Comment:
Place renderSections inside the div, its more legible

Reply:
u mean this?

`<div>{}</div>`